src/index.html

```diff
@@ -6,9 +6,73 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo-header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png"
+        class="moyo-logo"
+        alt="moyologo"
+        >
+      </a>
+
+      <nav class="moyo_nav">
+        <ul class="nav__list">
+          <li class="link-container">
+            <a class="nav__link is-active" href="">
+              Apple
+            </a>
+          </li>
+
+          <li class="link-container">
+            <a class="nav__link" href="">
+              Samsung
+            </a>
+          </li>
+
+          <li class="link-container">
+            <a class="nav__link" href="">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="link-container">
+            <a class="nav__link"
+            href=""
+            data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="link-container">
+            <a class="nav__link" href="">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="link-container">
+            <a class="nav__link" href="">
```

Comment:
```suggestion
            <a class="nav__link" href="#">
```
Не оставляй атрибут пустым